geomesa-quickstart-fsds/src/main/java/org/geomesa/example/fsds/FSDSQuickStart.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0 which
+ * accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ */
+
+package org.geomesa.example.fsds;
+
+import org.apache.commons.cli.ParseException;
+import org.geomesa.example.quickstart.GDELTData;
+import org.geomesa.example.quickstart.GeoMesaQuickStart;
+import org.geomesa.example.quickstart.QuickStartData;
+import org.geotools.data.DataStore;
+import org.geotools.data.DataUtilities;
+import org.locationtech.geomesa.fs.FileSystemDataStoreFactory;
+import org.locationtech.geomesa.fs.storage.common.*;
+import org.locationtech.geomesa.fs.storage.api.PartitionScheme;
+import org.opengis.feature.simple.SimpleFeatureType;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class FSDSQuickStart extends GeoMesaQuickStart {
+
+    // use gdelt data
+    private QuickStartData data = new GDELTData();
+
+    public FSDSQuickStart(Map<String, String> params) { super(params); }
+
+    public FSDSQuickStart(String[] args) throws ParseException {
+        super(args, new FileSystemDataStoreFactory().getParametersInfo());
+    }
+
+    @Override
+    public SimpleFeatureType createSchema(DataStore datastore, QuickStartData data) throws IOException {
+        SimpleFeatureType sft = data.getSimpleFeatureType();
```

Comment:
maybe we should break out `getSimpleFeatureType(QuickStartData data)` into a separate method we can override